sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs

```diff
@@ -1423,20 +1492,21 @@ public virtual async Task<Response<IList<CertificateContact>>> DeleteContactsAsy
         /// <summary>
         /// Merges a certificate or a certificate chain with a key pair currently available in the service. This operation requires the certificate/create permission.
         /// </summary>
-        /// <param name="certificateMergeOptions">The details of the certificate or certificate chain to merge into the key vault.</param>
+        /// <param name="mergeCertificateOptions">The details of the certificate or certificate chain to merge into the key vault.</param>
```

Comment:
Are options and details the same?

Reply:
We defined them early with @KrzysztofCwalina and @tg-msft that "Options" are input, and "Details" are output when read-only. "Properties" fall in between, basically, i.e. can be read-write and used for input and output.